skyportal/handlers/api/observation_plan.py

```diff
@@ -820,3 +820,57 @@ def delete(self, observation_plan_request_id):
             return self.error(f'TreasureMap delete failed: {request_text}')
         self.push_notification(f'TreasureMap delete succeeded: {request_text}.')
         return self.success()
+
+
+class ObservationPlanGeoJSONHandler(BaseHandler):
+    @auth_or_token
+    def get(self, observation_plan_request_id):
+        """
+        ---
+        description: Get GeoJSON summary of the observation plan.
+        tags:
+          - observation_plan_requests
+        parameters:
+          - in: path
+            name: observation_plan_id
+            required: true
+            schema:
+              type: string
+        responses:
+          200:
+            content:
+              application/json:
+                schema: SingleObservationPlanRequest
+        """
+
+        options = [
+            joinedload(ObservationPlanRequest.observation_plans)
+            .joinedload(EventObservationPlan.planned_observations)
+            .joinedload(PlannedObservation.field)
+            .undefer(InstrumentField.contour_summary)
+        ]
+
+        observation_plan_request = ObservationPlanRequest.get_if_accessible_by(
+            observation_plan_request_id,
+            self.current_user,
+            mode="read",
+            raise_if_none=True,
```

Comment:
Convert to check if no results then `self.error`

Reply:
good point